packages/core/src/workers/password-encryption-worker.ts

```diff
@@ -0,0 +1,36 @@
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import { findUp } from 'find-up';
+import { Tinypool } from 'tinypool';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
better to [bump the node version](https://www.sonarsource.com/blog/dirname-node-js-es-modules/#the-new-es-module-way) and use `import.meta.dirname`? (can do it later. also the docs is [out-dated](https://docs.logto.io/logto-oss/get-started-with-oss) which still shows node 18)

Reply:
add a TODO comment, will do in a separate PR.